ios/RNCallKeep/RNCallKeep.m

```diff
@@ -449,8 +477,10 @@ + (void)reportNewIncomingCall:(NSString *)uuidString
                      hasVideo:(BOOL)hasVideo
           localizedCallerName:(NSString * _Nullable)localizedCallerName
                   fromPushKit:(BOOL)fromPushKit
+                     resolver:(RCTPromiseResolveBlock)resolve
+                     rejecter:(RCTPromiseRejectBlock)reject
 {
-    [RNCallKeep reportNewIncomingCall: uuidString handle:handle handleType:handleType hasVideo:hasVideo localizedCallerName:localizedCallerName fromPushKit: fromPushKit payload:nil withCompletionHandler:nil];
+    [RNCallKeep reportNewIncomingCall: uuidString handle:handle handleType:handleType hasVideo:hasVideo localizedCallerName:localizedCallerName fromPushKit: fromPushKit payload:nil withCompletionHandler:nil resolver:resolve rejecter:reject];
 }
```

Comment:
Har vi brug for denne variant? Vi kalder den jo ikke fra displayIncomingCall. Jeg tænker, at for overskuelighedens skyld er fire overloads af samme metodenavn lige i overkanten.